go-chaos/internal/chaos-experiments/camunda-cloud/scaling/broker-scaling.json

```diff
@@ -69,7 +70,8 @@
                 "type": "process",
                 "path": "zbchaos",
                 "arguments": ["verify", "readiness"],
-                "timeout": 900
+                "timeout": 900,
+                "retryUntilTimeout" : "true"
```

Comment:
I would propose you make it part of the arguments and support this also in the CLI as new parameter, because otherwise we only have a feature now supporting for the worker but not for the cli which I would like to avoid. Especially for testing and trying out the cli is useful.

Reply:
Do we have to stick to chaostoolkit spec? :thought_balloon: 

Unfortunately adding the parameter in the command does not solve this problem. We already have a retry implemented in the command. The problem is that port-forwarding is not re-connected until the job is retried. We can only fail the job 3 times because after that it raises an incident and the test fails.

Reply:
Also right now, because the default job timeout is 5 minutes there are multiple workers running for the same job if the action takes longer than 5 minutes. For actions similar to scale up, it always takes more than 5 minutes. This is why I want to provide a more generic solution at the chaos worker.

Reply:
If we want to stick to the `chaostoolkit` spec, I think it would also make sense to set "retryUntilTimeout" as the default behaviour. Right now the commands implementation is not respecting the timeout specified anyway. Each command has its own way of handling retry and different timeout. Configured timeout is actually ignored at the command level.